workflow permissions (#102)
ASFHyP3/hyp3-cookiecutter: CHANGELOG.md

and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [0.5.1]

### Added
- Added `permissions` field for every GitHub Actions job in the cookiecutter template, to address https://github.com/ASFHyP3/actions/issues/220

### Changed
- Upgraded `ASFHyP3/actions` reusable actions to `v0.18.1`.

### Fixed
- Updated README instructions for creating a HyP3 plugin. In particular, added instructions for granting the GitHub user account (e.g. `tools-bot` for `ASFHyP3` repos) sufficient permissions for performing releases.

> - Changed the value of the `sync_pr_label` parameter for `reusable-release.yml` from `actions-bot` to `{{ cookiecutter.github_username }}` (e.g. `tools-bot` for `ASFHyP3` repos).

Reviewer: ```suggestion
- Removed the `sync_pr_label` parameter for `reusable-release.yml`  (will default to `tools-bot` for use with `ASFHyP3` repos).
```

Or maybe just don't mention it b/c of issue #103
